src/Cake.Core/Polyfill/AssemblyHelper.cs

```diff
@@ -26,7 +26,7 @@ public static Assembly LoadAssembly(AssemblyName assemblyName)
             return Assembly.Load(assemblyName);
         }
 
-        public static Assembly LoadAssembly(ICakeEnvironment environment, IFileSystem fileSystem, FilePath path)
+        public static Assembly LoadAssembly(ICakeEnvironment environment, IFileSystem fileSystem, ICakeLog log, FilePath path)
```

Comment:
Should we regard this as a breaking change?

Reply:
No, it's an internal class

Reply:
Sounds good to me!